src/reducers/map.jsx

```diff
@@ -46,6 +46,21 @@ export const processStateAggregations = agg => {
 // ----------------------------------------------------------------------------
 // Action Handlers
 
+/**
+ * Updates the state when an tab changed occurs, reset values to start clean
+ *
+ * @param {object} state the current state in the Redux store
+ * @returns {object} the new state for the Redux store
+ */
+export function handleTabChanged( state ) {
+  return {
+    ...state,
+    expandedTrends: [],
+    filterNames: [],
```

Comment:
I missed something. What are filters being reset  when the tabs vhange?

Reply:
these are just the parent rows from the row chart that are clickable

Reply:
That seems like a more appropriate name

Reply:
updated for clarity.  It was leftover code from Explore